sdk/turing/router/config/autoscaling_policy.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass, field
+from heapq import merge
+
+import turing.generated.models
+from turing.generated.model_utils import OpenApiModel
+
+
+@dataclass
+class AutoscalingPolicy:
```

Comment:
Would be nice to add some docstrings to the `AutoscalingPolicy` class, e.g to describe supported values for metric parameter. Even better, we could introduce Enums here?

Reply:
Done!

Reply:
Done! Thanks for the suggestion. :)